storage/index.go

```diff
@@ -0,0 +1,71 @@
+package storage
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	gethTypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/onflow/flow-go/fvm/evm/types"
+	"math/big"
```

Comment:
nit: partition by stdlib and external libs
```suggestion
	"math/big"

	"github.com/ethereum/go-ethereum/common"
	gethTypes "github.com/ethereum/go-ethereum/core/types"
	"github.com/onflow/flow-go/fvm/evm/types"
```

same comment for other files